pykafka/cluster.py

```diff
@@ -298,9 +301,9 @@ def get_offset_manager(self, consumer_group):
                 if i == MAX_RETRIES - 1:
                     raise
             except SocketDisconnectedError:
-                raise KafkaException("Socket disconnected during offset manager "
-                                     "discovery. This can happen when using PyKafka "
-                                     "with a Kafka version lower than 0.8.2.")
+                log.error("Socket disconnected during offset manager "
```

Comment:
If we pass over `SocketDisconnectedError` here, the next loop iteration will still be talking to `broker` which we selected in L287, so wouldn't that remain disconnected? I wonder if a `self.update()` would be in order, but not entirely sure if that's safe in the middle of this.